### ISSUE TITLE
Episode 5 "Getting the data": Loop over filenames rather than paths

### ISSUE THREAD

**Issue by klieret** [hsf-training/hsf-training-matplotlib#80]
```python

csvs = [
    pd.read_csv(
        "https://raw.githubusercontent.com/GuillermoFidalgo/Python-for-STEM-Teachers-Workshop/master/data/4e_2011.csv"
    ),
    pd.read_csv(
        "https://raw.githubusercontent.com/GuillermoFidalgo/Python-for-STEM-Teachers-Workshop/master/data/4mu_2011.csv"
    ),
    pd.read_csv(
        "https://raw.githubusercontent.com/GuillermoFidalgo/Python-for-STEM-Teachers-Workshop/master/data/2e2mu_2011.csv"
    ),
]
```

can be done much more elegant by taking a list of filenames and then using a list comprehension

**Comment by matthewfeickert** [comment]
This was resolved in PR #141 (but the right keywords weren't used so it wasn't autoclosed).

**Comment by klieret** [comment]
Thanks @matthewfeickert :)
